### ISSUE TITLE
Cannot run matchtigs

### ISSUE THREAD

**Issue by enricorox** [algbio/matchtigs#3]
Hi! I've installed the last version of `matchtigs` using mamba. 

Running `matchtigs` I obtain
> $ matchtigs
> --debug-print-graphtrue
>
> '?'

That may be is expected but I cannot run `--help` or `--version`
> $ matchtigs --help
> assigning values to '' wasn't recognized' which wasn't expected, or isn't valid

> $ matchtigs --version
> Found argument 'version
>
> matchtigs'

What I am doing wrong?
Thanks.

In case it helps
> $ uname -a
> Linux xxxxxxxxxxxx 4.18.0-492.el8.x86_64 #1 SMP Tue May 9 17:56:55 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

**Comment by sebschmi** [comment]
Hi! That is strange. I just tried to do the same via mamba. It installs version `1.5.5`. The output of the commands is then as expected.

I installed matchtigs with `mamba install -c conda-forge -c bioconda matchtigs`.

Then e.g. `matchtigs --version` outputs:
```
07:57:54 [INFO] Logging initialised successfully
matchtigs 1.1.0
```

Apparently, the version is not correct, but the program itself seems to work. The version not being correct is because the wrong version is hardcoded, and not because mamba would install anything wrong.
